updating/updating-restricted-network-cluster/uninstalling-osus.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="uninstalling-osus"]
+= Uninstalling the OpenShift Update Service from a cluster
+include::_attributes/common-attributes.adoc[]
+:context: uninstalling-osus
+
+toc::[]
+
+If you have a local copy of the OpenShift Update Service on your cluster, you can use the following procedures to uninstall it from the cluster.
+
+[id="update-service-delete-service"]
+== Deleting an OpenShift Update Service application
+
+You can delete an OpenShift Update Service application by using the {product-title} web console or CLI.
+
+include::modules/update-service-delete-service-web-console.adoc[leveloffset=+2]
+
+include::modules/update-service-delete-service-cli.adoc[leveloffset=+2]
+
+[id="update-service-uninstall"]
+== Uninstalling the OpenShift Update Service Operator
+
+To uninstall the OpenShift Update Service, you must first delete all OpenShift Update Service applications by using the {product-title} web console or CLI.
```

Comment:
Do you need _you must first delete all OpenShift Update Service applications_ here, as it is a prereq in the next sections? Just go with 

`You can delete the OpenShift Update Service (Operator?) by using the {product-title} web console or CLI.`